packages/renderer/src/lib/image/ImagesList.svelte

```
@@ -226,6 +226,51 @@ async function saveSelectedImages(): Promise<void> {
+}
+
+async function updateSelectedImages(): Promise<void> {
+  const selectedImages = images.filter(image => image.selected);
+  if (selectedImages.length === 0) {
+    return;
+  }
+
+  const originalStatuses = new Map(selectedImages.map(img => [img.id, img.status]));
+
+  bulkUpdateInProgress = true;
+  selectedImages.forEach(image => (image.status = 'UPDATING'));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)